src/ripple/app/ledger/impl/ConsensusImp.cpp

```diff
@@ -177,15 +177,22 @@ ConsensusImp::takePosition (int seq, std::shared_ptr<SHAMap> const& position)
     }
 }
 
-void
-ConsensusImp::visitStoredProposals (
-    std::function<void(LedgerProposal::ref)> const& f)
+std::vector <LedgerProposal::pointer>
```

Comment:
While you're in here, why don't you go ahead and get rid of these `pointer` and `ref` aliases?